src/views/AnalysisView.vue

```diff
@@ -156,50 +179,66 @@ const matrix = computed(() => {
             v-model="maxUtonalRoot"
           />
         </div>
-        <ChordWheel
-          class="chord-wheel"
-          type="utonal"
-          :maxChordRoot="maxUtonalRoot"
-          :virtualSynth="virtualSynth"
-          :width="500"
-          :height="400"
-          :lineWidth="2"
-          :backgroundRBG="[255, 255, 255]"
-          :fadeAlpha="fadeAlpha"
-          :shadowBlur="2"
-          strokeStyle="black"
-          textColor="blue"
-        />
       </div>
     </div>
   </main>
 </template>
 
 <style scoped>
-th {
+/* View */
+main {
+  padding: 1rem;
+  overflow-y: auto !important;
+}
+
+/* Interval matrix */
+.interval-matrix {
+  overflow-x: auto;
+  display: block;
+}
+.interval-matrix th {
   font-weight: bold;
 }
-table,
-th,
-td {
-  border: 1px solid;
+.interval-matrix table,
+.interval-matrix th,
+.interval-matrix td {
+  border: 1px solid var(--color-border);
 }
-table {
+.interval-matrix th, .interval-matrix td {
+  padding: 0.2rem 0.5rem;
+}
+.interval-matrix table {
   border-collapse: collapse;
+  text-align: center;
 }
 
+/* Content layout (medium) */
 div.columns-container {
-  height: 100%;
-  overflow-y: auto;
-  background-color: var(--color-border);
-  column-count: 1;
+  column-count: 2;
+  column-gap: 1rem;
+  overflow: hidden;
 }
 div.column {
-  background-color: var(--color-background);
-  overflow-x: hidden;
-  height: 100%;
+  overflow-y: auto;
 }
-div.analysis {
-  padding: 1rem;
+
+/* Content layout (medium and large) */
+@media screen and (min-width: 600px) {
+  div.columns-container {
+    column-count: 3;
+  }
+  div.column {
+    height: 100%;
+  }
+}
+
+/* Chord wheel */
+.chord-wheel {
+  border: 1px solid var(--color-border);
+  max-width: 100%;
+  height: auto;
+}
+[data-theme="dark"] .chord-wheel {
+  filter: invert(1);
```

Comment:
While nifty, I would prefer using the correct colors in the canvas itself. Even dark mode can use red to indicate otonality.

Reply:
I've removed this filter for now. Once the dark/light/default switch behavior is merged in, I think that will be a good time to come back to this